Included the secrets sync disable_strict_networking parameter in the important notes documentation (#840)
hashicorp/web-unified-docs: content/vault/v1.19.x/content/docs/updates/important-changes.mdx

To avoid issues and unexpected behavior with Vault Community, always provide a
valid `start_time` value when calling `/sys/internal/counters/activity` or
upgrade to v1.20.x.

> 

Reviewer: Is this really a known issue or does it belong under new behavior or breaking changes? It sounds like this is just how things work now and not something that we're going to "fix".

Also, if we introduced the functionality in 1.17.3, we need to update the 1.18 and 1.17 docs as well.

Author: it's a breaking change, for customers who are using a proxy to access the cloud provider it has triggered one sev-1 already.

> ### Secrets Sync SSRF Protection May Block Private Endpoints

Reviewer: ```suggestion
### Secrets sync SSRF protection may block private endpoints ((#ssrf-block-endpoint)
```
Style correction: use sentence case for headings, reserve capitalization for product and pseudo-product names
Formatting correction: use explicit anchors for important change headers

As of version 1.17.3, Vault's Secrets Sync includes additional Server-Side Request Forgery (SSRF) protection measures. This security enhancement prevents sync operations to certain IP ranges by introducing a new SSRF-safe HTTP client. The client specifically blocks requests to private IP ranges (such as 10.0.0.0/8), which affects users accessing cloud provider secret stores through private endpoints.

> Impact:

Reviewer: ```suggestion
```

> Secrets Sync operations to private IP ranges will be blocked
> Affects all destinations when accessed via private endpoints

Reviewer: ```suggestion
```
Folded into paragraph above

Example error message:

`couldn't sync secret with store: failed to publish event: dial tcp [IP]: prohibited IP address: [IP] is not a permitted destination (denied by: 10.0.0.0/8)`

> In Vault 1.19.x and newer the [disable_strict_networking](https://developer.hashicorp.com/vault/api-docs/system/secrets-sync#disable_strict_networking) configuration parameter has been added, which allows disabling both the IP address and port number restrictions used by the SSRF-safe sync clients.

Reviewer: ```suggestion
#### Recommendation

If you use private endpoints to sync secrets, set the
[`disable_strict_networking`](/vault/api-docs/system/secrets-sync#disable_strict_networking)
secret sync configuration parameter to `true` to disable IP address and port number restrictions
used by the SSRF-safe sync clients.
```
Since this page is part of the 1.19.x docs, the version number isn't needed. We can just tell them to set the configuration parameter if they need the associated functionality. For 1.17 and 1.18, the recommendation would be to upgrade to 1.19.x or higher so they can use the configuration parameter
